docker_templates/templates/docker_images/ros1_bridge_entrypoint.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 set -e
 
+unset ROS_DISTRO
```

Comment:
Nice workaround for the bridge image. Simplifies the other images for consistency and lets the entrypoint handle the uniqueness of the startup setup.